app.py

```diff
@@ -1,11 +1,43 @@
 import flask
+from flask import send_file
 
 
 app = flask.Flask(__name__)
-
 @app.route('/')
+def welcome():
+    return send_file('./templates/welcome.html')
+
+@app.route('/login')
+def login():
+    return send_file("./templates/login.html")
+
+@app.route('/signup')
+def signup():
+    return send_file("./templates/signup.html")
+
+@app.route('/logout')
+def logout():
+    return send_file("./templates/logout.html")
+    
+@app.route('/search')
 def search():
-    return 'Search Page'
+    return send_file("./templates/search.html")
+
+@app.route('/information')
```

Comment:
Please match the slug name with the actual html file name. 

Example: /info instead of /information since your html file is named info.html.

Reply:
This was a mistake on my end. Good catch.